PYME/IO/csv_flavours.py

```diff
@@ -202,7 +202,9 @@ def guess_flavour(colNames, delim=None, ext=None):
     for flavour in csv_flavours:
         if (not flavour == 'default') and all(idn in colNames for idn in csv_flavours[flavour]['idnames']):
             if fl is not None:
-                raise RuntimeError('Ambiguous flavour database: file matches both %s and %s' % (fl, flavour))
+                logger.info('Ambiguous flavour database: file matches both %s and %s' % (fl, flavour))
```

Comment:
I'm pretty sure that this runs within the error handling context manager, so the current `RuntimeError` results in the error message being visible to the user in a dialog. Demoting this to a `logger.info` seems unwise. I think that in most cases this is an error, and we should inform the user. Maybe use `PYME.warnings`? 

If we match more than one flavour, I'm not sure the most obvious thing is to ignore both the possible flavours and just go with the default - if we are doing this we should also explicitly tell the user that flavour detection failed and we are falling back on the default flavour.

Reply:
I agree it would be better to have a GUI response. What I find frustrating about the current approach is if flavor detection fails, PYMEVis raises this error and then refuses to open the file. Is there an easy way to throw this error in the context manager but then still display the `ImportTextDialog`?

Reply:
Yeah `warnings` is a good idea. Thanks!

Reply:
We could instead match the flavour with the larger number of correct columns. That may be a better default choice.